app/javascript/chat/userDetails.jsx

```diff
@@ -60,14 +60,11 @@ export default class UserDetails extends Component {
     const channel = this.props.activeChannel || {};
     const socialIcons = [];
     const userMeta = ['twitter_username', 'github_username', 'website_url'];
-    userMeta.forEach((metaProp) => {
+    userMeta.forEach(metaProp => {
       if (user[metaProp]) {
         let { className, hostUrl, srcImage, imageAltText } = userDetailsConfig[metaProp];
         socialIcons.push(
-          <a
-            href={`${hostUrl}${user[metaProp]}`}
-            target="_blank"
-          >
+          <a href={`${hostUrl}${user[metaProp]}`} target="_blank">
```

Comment:
Add the `rel="noopener"` attribute to links for security reasons. See https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
I made a simple codesandbox for a `<Link />` component. We could use something like this in Preact code so as to avoid having to remember to add the `rel` attribute being set properly. Thoughts?

https://codesandbox.io/s/priceless-turing-6mw5l

Reply:
Hmm I think an ESLint configuration is available or might be possible? If so, personally think that's good enough. I feel like an extra component might be additional complexity?

Reply:
Ahh, wasn't aware that there was a lint rule for it. Definitely do that then.

Reply:
Found it, so just popping it here for reference, https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md.

Reply:
Nice 👍